internal/platform/marvell-dpu.go

```diff
@@ -52,8 +52,11 @@ func (pi *MarvellDetector) IsDpuPlatform() (bool, error) {
 }
 
 func (pi *MarvellDetector) VspPlugin(dpuMode bool, vspImages map[string]string, client client.Client) *plugin.GrpcPlugin {
-	MarvellVspImage := vspImages["MarvellVspImage"]
-	return plugin.NewGrpcPlugin(dpuMode, client, plugin.WithVspImage(MarvellVspImage))
+	image := vspImages["MarvellVspImage"]
+	render_data := plugin.CreateCommonData(image)
+	render_data["Command"] = `[ "/vsp-mrvl" ]`
+	render_data["Args"] = `[ "--bridgeType=linux", "--interface=enp0s1f0d3", "--portMuxVsi=0x0e", "--ovsCliDir=/opt/p4/p4-cp-nws", "-v=debug" ]`
```

Comment:
These are (were) Intel specific arguments. Looking at Marvell's PR, these are not required: https://github.com/openshift/dpu-operator/commit/1b4275bb1b35367b0e9805aa9a6be0ad4cbfbea0#diff-f83f7283821d3bddf407b05738df9d57313ebc1ef62525d7a6ddabe16e192c3dR1